Add 413 response error for produce v3 api

trnguyencflt opened the pull request:
Follow up the PR https://github.com/confluentinc/rest-utils/pull/411, this is to update API doc for produce records v3 API in kafka-rest

trnguyencflt commented:
@tobydrake7 about the API doc change, do we need to anything more other than change this openapi.yaml file? Thanks 👍

tobydrake7 commented:
> @tobydrake7 about the API doc change, do we need to anything more other than change this openapi.yaml file? Thanks 👍

@trnguyencflt I believe this update to the API spec is all that is necessary, since it's imported into the public docs and is an error response description.

tobydrake7 commented on api/v3/openapi.yaml:
```diff
@@ -4493,6 +4495,20 @@ components:
                     </html>'
 
+    RequestEntityTooLargeErrorResponse:
+      description: 'This implies client is sending the request payload that is larger than the max message size the
+        server will accept.'
```
```suggestion
      description: 'This implies the client is sending a request payload that is larger than the maximum message size the
        server can accept.'
```

trnguyencflt replied:
done

tobydrake7 commented on api/v3/openapi.yaml:
```diff
@@ -4493,6 +4495,20 @@ components:
+              description: "Thrown by /records API if payload size exceeds the message max size"
+              value:
+                error_code: 413
+                message:  "The request included a message larger than the max message size the server will accept."
```
```suggestion
                message:  "The request included a message larger than the maximum message size the server can accept."
```

trnguyencflt replied:
done

tobydrake7 commented on api/v3/openapi.yaml:
```diff
@@ -4493,6 +4495,20 @@ components:
+                message:  "The request included a message larger than the max message size the server will accept."
+
     UnsupportedMediaTypeErrorResponse:
       description: 'This implies client is sending the request payload format in an supported format.'
```
```suggestion
      description: 'This implies the client is sending the request payload in a supported format.'
```

trnguyencflt replied:
done

trnguyencflt commented:
> > @tobydrake7 about the API doc change, do we need to anything more other than change this openapi.yaml file? Thanks 👍
> 
> @trnguyencflt I believe this update to the API spec is all that is necessary, since it's imported into the public docs and is an error response description.

@tobydrake7 I've addressed your comments, could you please have a look again? thanks 👍

msn-tldr commented on api/v3/openapi.yaml:
```diff
@@ -4493,8 +4495,22 @@ components:
+
     UnsupportedMediaTypeErrorResponse:
-      description: 'This implies client is sending the request payload format in an supported format.'
+      description: 'This implies the client is sending the request payload format in an unsupported format.'
```
good catch

trnguyencflt replied:
all credit to @tobydrake7 😄

msn-tldr reviewed (approved):
lgtm
